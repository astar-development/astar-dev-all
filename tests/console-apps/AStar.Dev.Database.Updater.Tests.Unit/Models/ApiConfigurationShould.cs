namespace AStar.Dev.Database.Updater.Models;

public sealed class ApiConfigurationShould
{
    [Fact]
    public void ReturnTheExpectedToString() =>
        new ApiConfiguration()
           .ToString()
           .ShouldMatchApproved();
}
