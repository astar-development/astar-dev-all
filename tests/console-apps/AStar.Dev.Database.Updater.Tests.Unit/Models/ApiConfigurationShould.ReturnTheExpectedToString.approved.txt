{
  "directories": [],
  "filesApiConfiguration": {
    "baseUrl": "https://not.set.com",
    "scopes": []
  }
}